src/wallet/rpcwallet.cpp

```diff
@@ -1637,7 +1637,7 @@ UniValue listsinceblock(const JSONRPCRequest& request)
             "\nGet all transactions in blocks since block [blockhash], or all transactions if omitted\n"
             "\nArguments:\n"
             "1. \"blockhash\"   (string, optional) The block hash to list transactions since\n"
-            "2. target-confirmations:    (numeric, optional) The confirmations required, must be 1 or more\n"
+            "2. target-confirmations:    (numeric, optional) Return the hash of the block that has target-confirmations of confirmations, for use in a further call to listsinceblock.\n"
```

Comment:
This is surely wrong. It should describe the meaning of the argument. And it does so in the master.